modules/apps/transfer/client/cli/tx.go

```diff
@@ -121,7 +133,42 @@ using the {packet-timeout-timestamp} flag. If no timeout value is set then a def
 	cmd.Flags().Uint64(flagPacketTimeoutTimestamp, defaultRelativePacketTimeoutTimestamp, "Packet timeout timestamp in nanoseconds from now. Default is 10 minutes. The timeout is disabled when set to 0.")
 	cmd.Flags().Bool(flagAbsoluteTimeouts, false, "Timeout flags are used as absolute timeouts.")
 	cmd.Flags().String(flagMemo, "", "Memo to be sent along with the packet.")
+	cmd.Flags().String(flagForwarding, "", "Forwarding information in the form of a comma separated list of portID/channelID pairs, denoting the intermediary hops. If forwarding is specified any memo set will be included in the forwarding information created.")
+	cmd.Flags().Bool(flagUnwind, false, "Flag to indicate if the coin should be unwound to its native chain before forwarding.")
+
 	flags.AddTxFlagsToCmd(cmd)
 
 	return cmd
 }
+
+// parseForwarding parses the forwarding flag into a Forwarding object or nil if the flag is not specified. If the flag cannot
+// be parsed or the hops aren't in the portID/channelID format an error is returned.
+func parseForwarding(cmd *cobra.Command) (types.Forwarding, error) {
+	var hops []types.Hop
+
+	forwardingString, err := cmd.Flags().GetString(flagForwarding)
+	if err != nil {
+		return types.Forwarding{}, err
+	}
+	if strings.TrimSpace(forwardingString) == "" {
+		return types.Forwarding{}, nil
+	}
```

Comment:
I think you'd need to use a different default instead of `""` then, right? in order to be able to return an error on empty string if the flag is there. Unless there's a cuter way to do it